### PR TITLE
Switch to OAuth Server-to-Server Authentication (instead of JWT)

### PR DESCRIPTION
**Pull request by gabeabrams**
Also removed support for HUIT Apigee

### CODE REVIEW

File: docBuilder/DOCS_HOME.md [hunk 1]
```diff
@@ -12,22 +12,27 @@ This project is designed _by_ educators _for_ educators.
...
-  secret: 'NM9UO1..',
+```ts
+import initZoomAPI from 'zaccl';
+
+const api = initZoomAPI({
+  zoomHost: 'harvard.zoom.us',
+  accountId: 'asd8049jtaihdf8',
+  clientId: 'adsna803948jr089j23t8sdfiu7',
+  clientSecret: '138fh8w7erik0a9g8d0f9ahs87ert872ye9rhuhaskudhf',
 });
```
Comment (karendolan) [inline docBuilder/DOCS_HOME.md:27]: Make these more dummy looking to avoid confusion for lib users.

File: docs/index.html [hunk 2]
```diff
@@ -67,17 +67,21 @@ <h1 style="display: flex; align-items: center;">
...
+<h2>1. Create an &quot;api&quot; instance</h2>
+<p>You can include credentials inline:</p>
+<pre class="prettyprint source lang-ts"><code>import initZoomAPI from 'zaccl';
+
+const api = initZoomAPI({
+  zoomHost: 'harvard.zoom.us',
+  accountId: 'asd8049jtaihdf8',
+  clientId: 'adsna803948jr089j23t8sdfiu7',
+  clientSecret: '138fh8w7erik0a9g8d0f9ahs87ert872ye9rhuhaskudhf',
 });
```
Comment (karendolan) [inline docs/index.html:79]: Make these more dummy looking

File: src/shared/helpers/sendZoomRequest.ts [hunk 3]
```diff
@@ -43,48 +39,20 @@ const sendZoomRequest = async (
...
 
-  if (usingHarvardApigee) {
-    // Using Harvard Apigee system
-    headers['X-Api-Key'] = token;
-  } else {
-    // Directly contacting Zoom
-    headers.Authorization = `Bearer ${token}`;
-  }
+  // Add token to request
+  headers.Authorization = `Bearer ${token}`;
```
Comment (karendolan) [inline src/shared/helpers/sendZoomRequest.ts:51]: Looks like no going back

**Review by karendolan — COMMENTED** [review]
Looks good to me. I just added some comments about the realistic looking Zoom init example params

File: src/shared/helpers/sendZoomRequest.ts (hunk 3 above)
Comment (gabeabrams) [reply src/shared/helpers/sendZoomRequest.ts:51]: I'm redoing this to allow for rolling back

**Comment by gabeabrams** [comment]
Instead of this PR, I've created a new PR that allows for all types of auth.

**Pull request closed without merging**